pymemcache/serde.py

```diff
@@ -15,8 +15,7 @@
 from functools import partial
 import logging
 from io import BytesIO
-import six
-from six.moves import cPickle as pickle
+import pickle
 
 try:
```

Comment:
This try catch seems more or less related to python version compatibility, I think that it can be adapted inline through along this module and then removed here.

Thoughts?

Reply:
done